### PR TITLE
add documentation on withAnyType method of error asserter

### LINKED ISSUE #153: error asserter : missing documentation on withAnyType

### PR DESCRIPTION
**Pull request by agallou**
fixes #153

### CODE REVIEW

**Comment by jubianchi** [comment]
:+1:

File: source/en/asserters/error.inc.rst [hunk 1]
```diff
@@ -114,6 +114,36 @@ withType
    ;
 
 
+.. _with-any-type:
+
+withAnyType
+===========
+
+``withAnyType`` does not check the type of the raised error. That's the default  behaviour. So ``->error()->withAnyType()->exists()`` is the equivalent of ``->error()->exists()``. This method is here if you want to add semantic to your test.
```
Comment (marmotz) [inline source/en/asserters/error.inc.rst]: double space between "default" and "behaviour"

File: source/en/asserters/error.inc.rst [hunk 2]
```diff
@@ -114,6 +114,36 @@ withType
...
+
+
+.. code-block:: php
+
+   <?php
```
Comment (marmotz) [inline source/en/asserters/error.inc.rst]: wrong indentation

**Comment by agallou** [comment]
@marmotz double space removed

**Comment by marmotz** [comment]
Thank you!

**Pull request merged**